blazor/treemap/legend.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 Legend is used to provide valuable information for interpreting what the TreeMap displays. The legends can be represented in various colors, shapes or other identifiers based on the data.
 
+The below video demonstrates the legend customization options in the Blazor TreeMap component.
```

Comment:
The below video demonstrates the customization of legend in Blazor TreeMap component.